src/devices/Pn5180/samples/Pn5180sample.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
+    </ProjectReference>
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Cannot put comment there but can you please replace PNG with JPEG so it's not 1.5MB 😄

Reply:
It's because there a transparent background. Transparent background are a recommendation now as more and more websites uses dark modes (docs.microsoft.com is now) and pictures are rendering correctly only without the background. That said, I can for sure adjust the size of the picture, no need to have a native 4K size.